ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/validator/preexecution/PreExecutingSubmissionValidatorSpec.scala

```diff
@@ -174,9 +209,11 @@ object PreExecutingSubmissionValidatorSpec {
   private final case class TestValue(value: Option[DamlStateValue])
 
   private object TestValue {
+
```

Comment:
```suggestion
```

Reply:
scalafmt did that on save, doesn't it for you?

Reply:
I don't have reformatting enabled on save. If the lint checks passed without these whitespaces before, then they are not really needed. The scalafmt settings IntelliJ uses must be aligned with what's being enforced by the lint checks (there seems to be a difference).

Reply:
So apparently there is, I'll rework my setup and remove these NLs.

Reply:
Done.